templates/index.html

```diff
@@ -6,6 +6,7 @@
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta/css/bootstrap.min.css" integrity="sha384-/Y6pD6FV/Vv2HJnA6t+vslU6fwYXjCFtcEpHbNJ0lyAFsXTsjBbfaDjzALeQsN6M" crossorigin="anonymous">
     <link href="static/css/custom.css" type="text/css" rel="stylesheet">
     <link href="static/css/chosen.css" type="text/css" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-datepicker/1.4.1/css/bootstrap-datepicker3.css"/>
```

Comment:
This date picker [doesn’t support bootstrap4 yet](https://github.com/uxsolutions/bootstrap-datepicker/issues/1670#issuecomment-329536391). I wonder if there’s one that does?

Reply:
If not… I wonder if there’s a case for using bootstrap3 instead? v4 is very newly out of alpha, and I am not sure it is terribly well-supported by these sorts of extensions yet.